DatadogRUM/Sources/RUM.swift

```diff
@@ -53,3 +53,11 @@ public enum RUM {
         rum.monitor.notifySDKInit()
     }
 }
+
+extension RUM {
+    /// Attributes that can be added to RUM calls that have special properies in Datadog.
+    public struct Attributes {
+        /// Add a custom fingerprint to the error in this log. Requires that the log be supplied with an Error.
```

Comment:
blocking/ similar to logs; also it calls "log" not RUM event. Ultimately:
```suggestion
        /// Add a custom fingerprint to the RUM error.
        /// The value of this attribute must be a `String`.
```